holmes/core/tools.py

```diff
@@ -157,6 +157,7 @@ def check_prerequisites(self):
             except subprocess.CalledProcessError as e:
                 self._enabled = False
                 self._disabled_reason = f"prereq check failed w/ errorcode {e.returncode}"
+                logging.exception(f"Toolset {self.name} : Failed to run prereq command {prereq}")
```

Comment:
Could you add a log for this elsewhere? We're deliberately not logging it here to keep the CLI output clean.

Reply:
fixed